### ISSUE TITLE
Ivy bridge bios tweaks:

### ISSUE THREAD

**Issue by bibikalka1** [bibikalka1/HP_Z420_Z620_Z820_BOOTBLOCK2013_BIOS_mod#2]
Placeholder

**Comment by ddr2020** [comment]
Not sure how I can help here... I have just one of those platforms as UP, use it for gaming and perhaps will phase it out soon. The Supermicro BIOS seems to be a bit more flexible as of settings; mainboards can be found cheap atm on ebay and they're easy to fit into a normal ATX case, with a decent power supply and accomodating mainstream GPUs. From the project page, seems ellaborate, IMO HP workstations are adequate for running decent home parallel workloads like rendering or virtualization, but I'm no longer into SMP. The lack of AVX2 and newer instruction set is seen in modern games as a severe clock-per-clock penalty, overclocking is limited and the platform shows its age.

**Comment by bibikalka1** [comment]
@ddr2020 

OK, thanks for the details!

I am using my Z620 as a casual 1080p gaming platform with RX 580. I think it has more than enough room for another graphics card upgrade with PCIE3 x16. Plus components such as DDR3 are dirt cheap too. I have an overclockable Xeon here that I might squeeze another 20% out of.

I agree Haswell is nicer, but I don't have that one handy, and don't want to spend much to acquire one.

So I am interested in implementing some useful features into BIOS for Ivy Bridge - bifurcation caught my eye. Do you know any other ones that are useful?

**Comment by ddr2020** [comment]
My guess is that you can pull some extra FPS from a UP Haswell Xeon than a MP Ivy Bridge. There are some H81/B85 mainboards there who can squeeze turbo boost out of a cheap E3 v3 and in my past I've gotten for example better fps from a UP E3 1231v3@3800MHz than from a stock E5 2680v2 in Horizon Zero Down on a GTX1070...
Bifurcation is implemented for Supermicro in latest BIOSes. Perhaps you can extract the dxe's but not sure how you will parametrize them.

**Comment by bibikalka1** [comment]
@ddr2020 

OK, thanks! I will take a look at that SuperMicro X9SRI-F BIOS. Not sure if HP had this implemented for ZX20 in 2019.

Regarding Haswell vs Ivy Bridge, I think Haswell is probably faster here and there, but not by much. In spec comparison of the 2 I did not spot any new must have features. So if one has a well configured Ivy Bridge - easier to keep that, instead of splurging some $ on a minor upgrade.

OF course, if the upgrade is getting someone 10 years into the future as in 10+ gen of Intel chips - that's a different story!

**Comment by ddr2020** [comment]
For RX580, you're right, the difference will be minimal if you stay at 1080p and you're already in the 60fps range. You may get the same boost or more by gpu overclocking, enabling RSR or using FSR3 in games.

**Comment by bibikalka1** [comment]
@ddr2020 

Yep, correct! And then one could grab a fancier video card instead of dumping the same budget on the new motherboard/cpu/memory before one can even talk about a new video card. I still plan to try to overclock a 1650v2 cpu to something like 4.3 Ghz - not quite the modern cpu, but good enough for a while.

The key question if the actual hardware lasts - capacitors don't try out, voltage regulators don't burn out, etc.
